src/test/java/org/sonatype/gradle/plugins/scan/common/DependenciesFinderTest.java

```diff
@@ -60,33 +61,82 @@ public void testFindResolvedDependencies_includeCompileDependencies() {
   }
 
   @Test
-  public void testFindResolvedDependencies_includeLegacyAndroidDependencies() {
+  public void testFindResolvedDependencies_includeRuntimeDependencies() {
+    Project project = buildProject(RUNTIME_CLASSPATH_CONFIGURATION_NAME, false);
+    Set<ResolvedDependency> result = finder.findResolvedDependencies(project, false);
+    assertThat(result).hasSize(1);
+  }
+
+  @Test
+  public void testFindResolvedDependencies_includeLegacyCompileAndroidDependencies() {
     Project project = buildProject("_releaseCompile", true);
     Set<ResolvedDependency> result = finder.findResolvedDependencies(project, false);
     assertThat(result).hasSize(1);
   }
 
   @Test
-  public void testFindResolvedDependencies_includeAndroidDependencies() {
+  public void testFindResolvedDependencies_includeLegacyRuntimeApkAndroidDependencies() {
+    Project project = buildProject("_releaseApk", true);
```

Comment:
minor: Some tests using the constant value while some tests using the String value (in this case `"_releaseApk"`) perhaps just one style ?

Reply:
> minor: Some tests using the constant value while some tests using the String value (in this case `"_releaseApk"`) perhaps just one style ?

There are constants available for ‘compileClasspath' and 'runtimeClasspath’ as they come for free in the Gradle API development toolkit, but there aren’t for the Android configurations like the one you mention